### ISSUE TITLE
How to use old v1 I m unable to add extention

### ISSUE THREAD

**Issue by mmani648** [ultrafunkamsterdam/undetected-chromedriver#592]
V2 is not helpful you can't add extention you can add proxies slenium wire use stupid certs which chrome don't trust than how to use it.i just want to know how to use old v1 or any way in this world to use extension in v2

**Comment by sebdelsol** [comment]
You can start by reading the documentation for how to use v1… And you’ll be back to raise another « issue » about why you’ve been detected.

Anyway Extensions work like a charm. Check #591.

Proxies work well too check #583.

And there’s an issue searchbox and it works very well too.

**Comment by mmani648** [comment]
issue number #583 wrong

**Comment by mmani648** [comment]
ptions.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
browser = webdriver.Chrome(  options=options)
not working

**Comment by QIN2DIM** [comment]
@mmani648 Maybe you should send the full code with the error message.

**Comment by mmani648** [comment]
no error msg it just dont loaded

**Comment by mmani648** [comment]
![image](https://user-images.githubusercontent.com/44944385/163516215-e48b9073-c409-4513-b623-9e808b1f7f4c.png)

**Comment by mmani648** [comment]
> @mmani648 Maybe you should send the full code with the error message.

can you show me live example it has been week i am unable to it :(

**Comment by QIN2DIM** [comment]
？？As far as I know the extension is a `.crx` suffix file

**Comment by QIN2DIM** [comment]
you can do this 

```python
options.add_extension(_extension_path)

```

**Comment by mmani648** [comment]
not working


> you can do this
> 
> ```python
> options.add_extension(_extension_path)
> ```

**Comment by mmani648** [comment]
they said we can't load extention i saw in one issue owner of repo said that but some  managed to do this 
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
but not working for me :(

**Comment by mmani648** [comment]
they said we can't load extention i saw in one issue owner of repo said that but some  managed to do this 
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
but not working for me :(

**Comment by mmani648** [comment]
they said we can't load extention I saw in one issue owner of repo said that but some  managed to do this 
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
but not working for me :(

**Comment by mmani648** [comment]
they said we can't load extention i saw in one issue owner of repo said that but some  managed to do this 
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
but not working for me :(

**Comment by mmani648** [comment]
they said we can't load extention I saw in one issue owner of repo said that but some  managed to do this 
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
but not working for me :(

**Comment by mmani648** [comment]
> you can do this
> 
> ```python
> options.add_extension(_extension_path)
> ```

please help please :(

**Comment by QIN2DIM** [comment]
It seems you don't know how to solve problems by the community .

**Comment by sebdelsol** [comment]
@mmani648, as stated before it works like a charm but you do it wrong.
* you've to be sure you give the directory where the manifest file lies.
* _Spaces in command line switches are interpreted as separators between switches so remove those_.
```python
# this won't work, it'll be interpreted as 3 weird switches
options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')

# this works
options.add_argument('--load-extension=C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
```
If you took the time you would have found it's already answered [here](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/586#issuecomment-1094933690).
Unfortunately the issues database is now hard to parse because it's littered with "issues" like yours. How ironic...

**Comment by mmani648** [comment]
> @mmani648, as stated before extensions works like a charm but you do it wrong.
> 
> * you've to be sure to give the directory where the manifest file lies : _it seems you've covered this one_.
> * _Spaces in command line switches are interpreted as separators between switches : remove those_.
> 
> ```python
> # this won't work, it'll be interpreted as 3 malformed switches, hence doing nothing
> options.add_argument('--load-extension = C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
> 
> # this works
> options.add_argument('--load-extension=C:/Users/Nc/Desktop/coinsnipelast/proxy_auth_plugin')
> ```
> 
> btw it's already been answered [here](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/586#issuecomment-1094933690).
> 
> You'll reckon it was not worth raising an issue for that.

working :) thanks
